cluster/validate-cluster.sh

```diff
@@ -32,15 +32,24 @@ attempt=0
 while true; do
   # The "kubectl get nodes" output is three columns like this:
   #
-  #     NAME                     LABELS    STATUS
-  #     kubernetes-minion-03nb   <none>    Ready
+  #     NAME                     LABELS    STATUS   AGE
+  #     kubernetes-minion-03nb   <none>    Ready    10m
   #
   # Echo the output, strip the first line, then gather 2 counts:
   #  - Total number of nodes.
   #  - Number of "ready" nodes.
   "${KUBE_ROOT}/cluster/kubectl.sh" get nodes > "${MINIONS_FILE}" || true
   found=$(cat "${MINIONS_FILE}" | sed '1d' | grep -c .) || true
-  ready=$(cat "${MINIONS_FILE}" | sed '1d' | awk '{print $NF}' | grep -c '^Ready') || true
+
```

Comment:
@jlowdermilk - Thanks for the hint. But I'm not going to use '--template' flag in the command for it is bound with api-version. I'm planning to use only '--api-version' and dump all output, then for different version, we can define different logic. I think it is good for further extension, maybe we'll have v2 or v3 later.